Dockerfile

```diff
@@ -43,4 +42,4 @@ ADD --chown=snyk:snyk . .
 # Complete any `prepare` tasks (e.g. typescript), as this step ran automatically prior to app being copied
 RUN npm run prepare
 
-ENTRYPOINT ["/usr/bin/dumb-init", "--", "bin/start"]
+ENTRYPOINT ["bin/start"]
```

Comment:
what about dumb-init?  does the UBI have something similar?

Reply:
Yes, UBI comes with its own init manager 🙏 
`ubi-minimal` doesn't so that's why I opted for the full `ubi`; would've been nice to have the slimmer image

Reply:
does UBI's init manager has the same capabilities of dumb-init? in terms of proxying signals etc